CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ JIRA REST API Client is an open source, community-driven project. If you'd like
 ## Working with Pull Requests
 1. Create your feature addition or a bug fix branch based on __`master`__ branch in your repository's fork.
 2. Make necessary changes, but __don't mix__ code reformatting with code changes on topic.
-3. Add entry in `CHANGELOG.md` file following http://keepachangelog.com/ format (if applicable).
+3. Add entry in `CHANGELOG.md` file following https://keepachangelog.com/en/1.0.0/ format (if applicable).
```

Comment:
I'd rather not add `en/1.0.0/` part, because it seems internal to website building engine and version itself might change in the future.

Reply:
As I see it, it is version of the specification, your project guidelines may not be compatible with future 2.0.0, so redirection to "latest" is not good in my opinion.

Reply:
I see. That makes sense now. Thank you.